Automattic/wp-calypso
# Newsletter: add byline settings

This PR adds the newsletter settings for the newsletter byline for .com users. 

## Proposed Changes

Before:
Section doesn't exit. 

After: 
<img width="772" alt="Screenshot 2024-09-05 at 6 40 53 AM" src="https://github.com/user-attachments/assets/9c04f2ab-165d-4353-b82a-5c9e488b703a">




## Why are these changes being made?

* To have feature parities with Jetpack. 

## Testing Instructions

* Navigate to /settings/newsletter/yoursite.com and notice that you can now toggle different settings. 
* And they work as expected. 

## Pre-merge Checklist
